django_celery_results/models/generic.py

```diff
@@ -0,0 +1,78 @@
+"""Database models."""
+
+import json
+
+from celery.result import GroupResult as CeleryGroupResult
+from celery.result import result_from_tuple
+from django.conf import settings
+from django.db import models
+from django.utils.translation import gettext_lazy as _
+
+from django_celery_results.models.abstract import (
+    AbstractGroupResult,
+    AbstractTaskResult,
+)
+
+
+class TaskResult(AbstractTaskResult):
+    """Task result/status."""
+
+    class Meta(AbstractTaskResult.Meta):
+        """Table information."""
+
+        abstract = False
+        app_label = "django_celery_results"
+
+
+class ChordCounter(models.Model):
```

Comment:
why we are not using any abstract model here?

Reply:
There is no specific reason at the moment. Will be added in the next iteration/update